rust/kcl-lib/tests/kcl_samples/sheet-metal-bracket/artifact_commands.snap

```diff
@@ -181,7 +181,7 @@ description: Artifact commands sheet-metal-bracket.kcl
         "type": "line",
         "end": {
           "x": 0.0,
-          "y": 2.4676,
+          "y": 2.6529,
```

Comment:
Coordinates changed on this model here. The above change could be a conversion to relative, but this isn't.